client/components/checklist/task.js

```diff
@@ -196,6 +199,11 @@ class Task extends PureComponent {
 									>
 										{ taskActionButtonText }
 									</Button>
+									{ ! completed && showSkip ? (
+										<Button className="checklist__task-skip" onClick={ onDismiss }>
+											Skip
```

Comment:
Should this be translatable?

Reply:
Thanks for noticing this @Aurorum 

`'Skip'` is already [translated](https://translate.wordpress.com/projects/wpcom/-all-translated/35472/). Just need to wrap it in a `translate` function.

Reply:
It definitely should. Thanks for the suggestion. I updated the code.